application/classes/Ushahidi/Validator/Post/Create.php

```diff
@@ -280,6 +282,7 @@ public function checkValues(Validation $validation, $attributes, $fullData)
 	 */
 	public function checkStageInForm(Validation $validation, $completed_stages, $fullData)
 	{
+		$completed_stages = !empty($fullData['completed_stages']) ? $fullData['completed_stages'] : [];
```

Comment:
I don't think this is needed. We only need to validate the stages themselves when `completed_stages` change.